### DISCUSSION TITLE
How to push commit to GitHub with ssh?

### DISCUSSION THREAD

**Discussion by qyl27** [community/community#23924] [New to GitHub]
<p>Please excuse my stupid, but after GitHub removed the push method with password, how to push commit  with ssh?</p>
<p>I am using Tortoise Git.<br>
And tried to add below to my <code>.gitconfig</code> file.</p>
<pre><code class="lang-auto">[url "git://git@github.com:"]
  insteadOf = https://github.com/
</code></pre>
<p>Then I generated a ssh keypair, and uploaded to my github settings.</p>
<p>But when I tried to push my commit, Git shows a fatal error: unable to look up <a href="mailto:git@github.com">git@github.com</a> (port 9418) ().</p>
<p>So, where is my mistake?</p>

**Comment by airtower-luna** [comment, marked as answer]
<aside class="quote no-group" data-username="AmemiyaSigure" data-post="1" data-topic="195462">
<div class="title">
<div class="quote-controls"></div>
 AmemiyaSigure:</div>
<blockquote>
<p>So, where is my mistake?</p>
</blockquote>
</aside>
<p>The <code>git://</code> part, the SSH URL should start with just <code>git@github.com</code>. Note the port: SSH should be using 22, which indicates Git misunderstood the protocol it’s supposed to use.</p>

**Comment by qyl27** [comment]
<p>Oh, it works, thank you a lot.</p>

**Comment by sudo-nautilus** [comment]
<p>If it works, you may mark it as the solution :cat:<br>
It may help others too.</p>
